framework/decode/vulkan_replay_consumer_base.cpp

```diff
@@ -2377,17 +2390,25 @@ VulkanReplayConsumerBase::OverrideCreateInstance(VkResult original_result,
 
         if (replay_create_info->ppEnabledExtensionNames)
         {
+            // If a WSI was selected on the command line we need to add that WSI surface extension name to the
+            // VkInstance
+            assert(application_);
+            auto wsi_context       = application_->GetWsiContext();
+            auto window_factory    = wsi_context ? wsi_context->GetWindowFactory() : nullptr;
+            auto cli_wsi_extension = window_factory ? window_factory->GetSurfaceExtensionName() : nullptr;
+            if (cli_wsi_extension)
+            {
+                filtered_extensions.push_back(cli_wsi_extension);
+            }
+
             // Swap the surface extension supported by platform the replay is running on if different from trace
```

Comment:
Is this comment still accurate? It looks like the swap may have been removed.

Reply:
Removed